Source/WebKit/Shared/WebEventConversion.cpp

```diff
@@ -81,7 +81,7 @@ class WebKit2PlatformMouseEvent : public WebCore::PlatformMouseEvent {
             break;
         case WebEventType::MouseForceUp:
             m_type = WebCore::PlatformEvent::Type::MouseForceUp;
-            m_force = WebCore::ForceAtForceClick;
```

Comment:
Surprised we were putting a value of 2 here! Is the `ForceAtForceClick` variable even correct? Why was it set to 2? The spec says the range is [0, 1]

Reply:
@nt1m this was introduced in [163047@main](https://commits.webkit.org/163047@main), before the PointerEvents spec was authored, so it makes sense for these values to not match the spec. `ForceAtClick` conveniently aligns with the pressure ranged defined in the specification and has a meaningful name, so I'm reusing it in this context.

Reply:
Should we remove `ForceAtForceClick` ?

Reply:
As we discussed offline, `ForceAtForceClick` is consequential in some downstream touch event handling code. Having said that, it _is_ a bit confusing since it's not a valid `PointerEvent.pressure` value, so instantiating a PointerEvent from a MouseForceDown event can put is in a state that does not conform to the PointerEvent spec.

To address this potential error while keeping the touch codepath happy, we choose to clamp incoming pressure values in `PointerEvent` instantiation.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/62ccdbdccfcf91cd3e56ea972a13f86c8e0eb1a2